data_format.md

```diff
@@ -133,3 +151,60 @@ AGENT_DTYPE = [
     ("label_probabilities", np.float32, (len(LABELS),)),
 ]
 ```
+
+### Traffic Light Faces
+Note: we refer to traffic light bulbs (e.g. the red light bulb of a specific traffic light) as `faces` in L5Kit.
+For the full list of available types for a bulb please consult our [protobuf map definition](https://github.com/lyft/l5kit/blob/20ab033c01610d711c3d36e1963ecec86e8b85b6/l5kit/l5kit/data/proto/road_network.proto#L615).
+
+Our semantic map holds static information about the world only. This means it has a list of all traffic lights, but not information about how their status changes over time.
+This dynamic information is instead stored in this array.
+Each array's element has a unique id to link it to the semantic map, a status (if `>0` then the face is active) and a reference to its parent traffic light.
```

Comment:
Could think of giving a concrete example here, ie >0 means red bulb is one, =0 for green bulb